### PR TITLE
Clarify the installation of EPEL repository on CentOS vs RHEL

### PR DESCRIPTION
**Pull request by widhalmt**
Some users reported the installation of the EPEL repository was unclear, especially for RHEL, so I tried to fix that.

I did not add CentOS 8 and  RHEL 8 for now because we still miss some packages there.

### CODE REVIEW

**Comment by dnsmichi** [comment]
Following the URL for RHEL really is that hard and manually install the rpms from their docs?

Anyhow, the commands for installing the packages needs to be put **after** the subscription manager thingy, as this is a required people always forget.

**Comment by widhalmt** [comment]
It's not that hard to follow the link, but since we put the CentOS part into the documentation, some users seem to be confused. They are used to having all things work the same in CentOS and RHEL and that's one of the view differences. So either replace the whole part with the link or split it up. I'd opt for the later and change my pull request.

I'll just update the request, AFAIK GitHub know allows for automatic squashing, right?

**Comment by dnsmichi** [comment]
I prefer PRs being squashed by the user, since squashing only works without creating a merge commit linking to the PR. We don't use that functionality via GitHub web there and squash on the CLI manually.

In terms of documenting everything - agreed, but if things are too simple users stop on the "why" and just copy-paste. That's not the intention behind the docs.

**Comment by dnsmichi** [comment]
For now, thanks.
